Sources/SpeziScheduler/Notifications/SchedulerNotifications.swift

```diff
@@ -24,6 +24,48 @@ import struct SwiftUI.AppStorage
 ///
 /// Notifications can be automatically scheduled for Tasks that are scheduled using the ``Scheduler`` module. You configure a Task for automatic notification scheduling by
 /// setting the ``Task/scheduleNotifications`` property.
+/// 
+/// When notifications are enabled, the app will send alerts based on the task’s schedule. 
+///     If you want to stop notifications for a specific event, you can call ``Event/complete()`` to mark the event as complete.
+///     This will prevent notifications only for that particular occurrence, while other events will still trigger notifications as scheduled.
```

Comment:
```suggestion
/// If you want to stop notifications for a specific event, you can call ``Event/complete()`` to mark the event as complete.
/// This will prevent notifications only for that particular occurrence, while other events will still trigger notifications as scheduled.
```